common/config/build-definitions/playground-master.yaml

```diff
@@ -0,0 +1,19 @@
+resources:
+- repo: self
+pool:
+  vmImage: Hosted
+steps:
+- task: NodeTool@0
+  displayName: 'Use Node 10.x'
+  inputs:
+    versionSpec: 10.x
+    checkLatest: true
+- script: 'node common\scripts\install-run-rush.js install --bypass-policy'

```

Comment:
We need to remove `--bypass-policy`.  Our repos should be demonstrating "best practices" for Rush.